app/assets/stylesheets/styles/components/guides/components/_small_screen_show_top.css.scss

```diff
@@ -0,0 +1,17 @@
+.small-guide {
+  .canopy {
+    height: 400px;
+  }
+
+  .guide-name {
+    @include background-gradient(rgba(0,0,0,0.85), rgba(0,0,0,0));
```

Comment:
`0.85` should be written without a leading zero as `.85`<br>Commas in function arguments should be followed by a single space